Makefile

```diff
@@ -14,7 +14,7 @@ LIBAV = `pkg-config --libs --cflags libswscale libavutil`
 LIBS  =  -lspeex -lasound -lpthread -lm
 JPEG  = -lturbojpeg
 SRC   = src/connection.c src/settings.c src/decoder*.c src/av.c src/usb.c src/queue.c
-USBMUXD = -lusbmuxd-2.0
+USBMUXD = -lusbmuxd
```

Comment:
This reverts the first commit to use -2.0. Does that mean it's just a matter of using `usbmuxd_send/recv` to fix the connection reset problem?

Looking at the implementation of those functions, it's just a matter of the socket being nonblocking. I'm surprised the problem only shows up on Arch in that case 🤔

Can we just remove the NONBLOCK flag from the usbmuxd socket and leave everything as-is?
Ot better yet, convert all other sockets to NONBLOCK and not have the av threads locked in syscalls (would also address #180).

Reply:
actually I didn't know how usbmuxd does its thing. I'll look into it and implement your suggestion if it works :)